### ISSUE TITLE
Impossible to repost an ad that was manually posted

### ISSUE THREAD

**Issue by dudududodododedede** [jackm/kijiji-manager#30]
When manually posting using an xml file (i.e. test.xml) there is no new xml file created (i.e. {id}.xml)

So when using the "repost" feature, this error gets thrown:

```
Cannot repost, ad file {id}.xml does not exist
```

**Comment by prgrm** [comment]
Yes same problem here

**Comment by dudududodododedede** [comment]
Added a fix for it here: https://github.com/jackm/kijiji-manager/pull/31

**Comment by jackm** [comment]
Good catch. I had completely forgotten about that step and how it is necessary for ad reposting to function.
